### PR TITLE
Feature/add azure storage

### PR DESCRIPTION
**Pull request by AlenaGavriluk**
image storage
file type and file size limitation on input
permission for images and tags in security config

### CODE REVIEW

File: frontend/src/components/CreatePostForm/CreatePostForm.tsx [hunk 1]
```diff
@@ -16,7 +17,13 @@ interface ICreatePostFormProps {
 
 const CreatePostForm: React.FC<ICreatePostFormProps> = ({ form, setForm, sendImage, allTags }) => {
   const { getRootProps, getInputProps } = useDropzone({
-    onDrop: files => sendImage({ file: files[0], inContent: true })
+    onDrop: files => {
+      if (files[0].size > (1024 * 1024)) {
```
Comment (Quipex) [inline frontend/src/components/CreatePostForm/CreatePostForm.tsx:21]: no magic numbers, use constants

File: frontend/src/components/CreatePostForm/CreatePostForm.tsx [hunk 2]
```diff
@@ -16,7 +17,13 @@ interface ICreatePostFormProps {
...
+        toastr.error('Error', 'File is too large, use image less then 1Mb');
```
Comment (Quipex) [inline frontend/src/components/CreatePostForm/CreatePostForm.tsx:22]:
```suggestion
        toastr.error('Error', 'File is too large, use image less than 1Mb');
```

File: frontend/src/components/CreatePostForm/CreatePostForm.tsx [hunk 3]
```diff
@@ -28,6 +35,10 @@ const CreatePostForm: React.FC<ICreatePostFormProps> = ({ form, setForm, sendIma
         }
       });
     } else {
+      if (event.target.files[0].size > (1024 * 1024)) {
+        toastr.error('Error', 'File is too large, use image less then 1Mb');
```
Comment (Quipex) [inline frontend/src/components/CreatePostForm/CreatePostForm.tsx:39]: logic duplication

**Review by Quipex — CHANGES_REQUESTED** [review]

**Review by kopot-o-d — APPROVED** [review]

**Review by Quipex — APPROVED** [review]
Lgtm

**Pull request merged**